### ISSUE TITLE
v3-consideration, low-importance: Add julia-repl syntax highlighting mode

### ISSUE THREAD

**Issue by SaschaMann** [exercism/exercism#5393]
It's common to write Julia code examples as if they were typed into a REPL including the returned value. For simple examples like 

```julia
julia> preptime(4)
8
```

the default julia syntax highlighting works okayish.

But for longer examples it adds random colours to the output and becomes somewhat messy:

```julia
julia> @testset "Arithmetics" begin
           @testset "Addition" begin
               @test 1  + 1 ==  2
               @test 40 + 2 == 42
           end
           @testset "Multiplication" begin
               @test 3 * 3 == 9
               @test 1 * 1 == 2 # this test will fail
               @test 3 * 0 == 0
           end
       end
Multiplication: Test Failed at REPL[15]:8
  Expression: 1 * 1 == 2
   Evaluated: 1 == 2
Stacktrace:
[...]
Test Summary:    | Pass  Fail  Total
Arithmetics      |    4     1      5
  Addition       |    2            2
  Multiplication |    2     1      3
ERROR: Some tests did not pass: 4 passed, 1 failed, 0 errored, 0 broken.
```

Julia docs generally use a second ```julia-repl syntax highlighting mode that only applies highlighting to the actual code, rather than the REPL prompt and output.

It would be nice to have support for that in the V3 UI.

**Comment by iHiD** [comment]
Where would this go? Markdown files (as above)? or somewhere else?

**Comment by SaschaMann** [comment]
Markdown files. `introduction.md` files for example: https://github.com/exercism/v3/blob/master/languages/julia/exercises/concept/functions-introduction/.docs/instructions.md (obv. that particular one is trivial so it doesn't matter)

**Comment by SaschaMann** [comment]
Using `highlight.js` would give us this feature for free, and also simplify doctests (https://github.com/exercism/v3/pull/2666): https://github.com/highlightjs/highlight.js/blob/21b146644e15ba2b101341da0d3e4dc61db53056/src/languages/julia-repl.js#L29-L38

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂

**Comment by SaschaMann** [comment]
AFAICT this is still relevant and should be reopened.

**Comment by joshgoebel** [comment]
**Re: Highlight.js**

We have `julia-repl` for this, but it is NOT a drop-in replacement for `julia` as it will ONLY highlight julia code that is part of a `julia>` block... so you can't just randomly mix and match code and repl output. You should already be able to use it in markdown snippets though unless Exercism just isn't including it for some reason?

**Comment by SaschaMann** [comment]
> You should already be able to use it in markdown snippets though unless Exercism just isn't including it for some reason?

Right, I forgot its included by default. I'll try if it works. Thanks for the heads up!
